velox/docs/spark_functions.rst

```diff
@@ -55,30 +56,33 @@ for :doc:`all <functions/spark/coverage>` functions.
     :widths: auto
     :class: rows
 
-    ================================  ================================  ================================  ==  ================================
```

Comment:
Coverage maps should be updated in a separate PR, not mixed with other changes.

Reply:
Get it, thanks for the info.